client/wordpress-com.js

```diff
@@ -157,7 +150,7 @@ const sections = [
 		name: 'themes',
 		paths: [ '/design' ],
 		module: 'my-sites/themes',
-		enableLoggedOut: config.isEnabled( 'manage/themes/logged-out' ),
+		enableLoggedOut: true,
```

Comment:
Correct. We should also remove `manage/themes/logged-out` from the config files if we set this to true, in case anyone tries to turn that flag off in the future expecting it to have any effect.